libkbfs/journal_server.go

```diff
@@ -18,8 +18,43 @@ import (
 	"golang.org/x/net/context"
 )
 
+// TODO: Expand this list to the whole team.
+var journalingBetaList = map[keybase1.UID]bool{
+	"ef2e49961eddaa77094b45ed635cfc00": true, // Jeremy Stribling, "strib"
+	"ebbe1d99410ab70123262cf8dfc87900": true, // Fred Akalin, "akalin"
+}
+
 type journalServerConfig struct {
+	// EnableAuto, if true, means the user has explicitly set its
+	// value. If false, then either the user turned it on and then
+	// off, or the user hasn't turned it on at all.
 	EnableAuto bool
+
+	// EnableAutoSetByUser means the user has explicitly set the
+	// value of EnableAuto (after this field was added).
+	EnableAutoSetByUser bool
+}
+
+func (jsc journalServerConfig) getEnableAuto(currentUID keybase1.UID) (
```

Comment:
Doesn't quite apply here, since `jsc` doesn't have a lock -- but added a similar function to `JournalServer` named `getEnableAutoLocked`